src/sass/components/_tour.scss

```diff
@@ -37,14 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  left: -($spacing-xx-small);
+  top: -($spacing-small);
+
+  .epic-progress & {
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
-    top: unset;
+    top: $spacing-small;
+    left: $spacing-large;
+  }
+
+  .metecho-breadcrumb & {
+    left: $spacing-large;
+    top: -($var-spacing-x-small);
+  }
+
+  .project-placeholder & {
+    top: $spacing-medium;
+  }
+
+  .project-list & {
+    left: $spacing-none;
+    top: $spacing-xxx-small;
   }
 
   .page-title & {
-    top: 25px;
+    left: $spacing-large;
+    top: $var-spacing-medium;
+  }
+
+  .user-info & {
+    left: -($spacing-xx-small);
+    top: -($spacing-small);
+  }
+
+  .page-options & {
+    left: -($spacing-xx-small);
+    top: -($spacing-medium);
+  }
+
+  .vertical-right & {
+    right: -($spacing-x-small);
+    top: -($spacing-medium);
+  }
+
+  .submit-btn & {
+    right: -($spacing-xxx-small);
+    top: -($spacing-large);
+  }
+
+  .task-create-btn & {
+    top: -($var-spacing-large);
+  }
+
+  .heading & {
+    top: -($spacing-medium);
+    left: -($spacing-xx-small);
```

Comment:
@dvdherron I think this is default, and not needed here? If it is needed, then we will want to reorder properties but I think we can remove the left: ...

Reply:
yes. the `left` property was redundant. removed.